src/components/EmptyState/EmptyState.scss

```diff
@@ -77,7 +77,7 @@ $centered-illustration-width: rem(226px);
   }
 
   .DetailsContainer {
-    padding-top: spacing();
+    padding-top: 0;
   }
```

Comment:
Are we able to remove this section all together? or is the setting the padding to zero needed to reset something else?